### ISSUE TITLE
Czech Republic missing geolocation

### ISSUE THREAD

**Issue by adamjstewart** [maja601/EuroCrops#38]
The Czech Republic shapefiles in v10 seem to be missing geolocation information.

### Steps to Reproduce

```console
> curl -LO https://zenodo.org/records/10118572/files/CZ_2023.zip
> unzip CZ_2023.zip 
> ogrinfo -so CZ_2023/CZ_2023.shp CZ_2023
...
Layer SRS WKT:
(unknown)
```

**Comment by xchan011** [comment]
The original source file was already missing the information, the reply we got upon inquiry is EPSG:5514. You can check our wiki page for Czech for more information.

**Comment by adamjstewart** [comment]
If that's the case, can the CRS be added to those files?

**Comment by xchan011** [comment]
In general, we try to minimize any changes to the source file apart from adding the harmonization. That being said, I just checked Czech's website and they have recently released a proper crop declaration shape file with CRS included (previously I had to merge the declarations from csv file with the LPIS shape file). So a new update with the official shape file (including the CRS) will be coming in the future.
